<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiberalMainOption</class>
 <widget class="QMainWindow" name="LiberalMainOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;
background-color: rgb(255, 208, 142);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_main_liberal_option_title">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>41</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>핵심교양</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_main_liberal_ment">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="text">
     <string>이미 수강하신 핵심교양을 선택하세요. </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right_arrow">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_right_arrow{
background-color: transparent;
border-image: url(:right_arrow3.png);
background: none;
background-repeat: none;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>691</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_challenge">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="title">
        <string>도전</string>
       </property>
       <widget class="QLabel" name="label_challenge">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>651</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>배달의민족 주아</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>21세기글로벌리즘과 한국, 21세기 기업 경영, 건강한 삶, 경제와 사회, 글로벌리즘과 세계경제, 
내 몸의 구조와 기능, 미래 사회와 소프트웨어, 성의과학, 소프트웨어 코딩과 적용, 우주의 이해, 
창업계획, 환경과 인간</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_communication">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="title">
        <string>소통</string>
       </property>
       <widget class="QLabel" name="label_communication">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>651</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>배달의민족 주아</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>공감프레젠테이션, 독일어(초급), 러시아어(초급), 불어(초급), 스페인어(초급), 아카데믹한국어2, 
일본어(초급), 중국어(초급), 중앙인과 한국사회</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_trust">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="title">
        <string>신뢰</string>
       </property>
       <widget class="QLabel" name="label_trust">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>651</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>배달의민족 주아</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>글로벌 한자, 동양철학의 세계, 언어와 의사소통, 역사란 무엇인가, 인간행동과 심리, 철학의 세계, 
프로젝트 글쓰기, 한국의 전통문화유산, 현대사회와 법, 현대사회와 언어, 현재사회의 쟁점</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_unite">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="title">
        <string>융합</string>
       </property>
       <widget class="QLabel" name="label_unite">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>651</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>배달의민족 주아</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>과학기술과 현대사회, 미래사회와 미디어, 생활 속의 생명과학, 수학의 세계, 시적인간의 세계읽기, 
식물과 문명, 식품과 건강, 인간과 미생물, 통계와 사회</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_creative">
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="title">
        <string>창의</string>
       </property>
       <widget class="QLabel" name="label_creative">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>651</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>배달의민족 주아</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>글로벌시대의 다문화콘텐츠, 다빈치명저읽기, 다빈치스포츠, 다빈치예술여행, 
문학과 예술의 사회사, 문학의 이해, 문화콘텐츠 입문, 생물의 기원과 진화, 
세계문화지리아카데미, 음악의 이해와 감상, 현대미술비평</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_borderline">
    <property name="geometry">
     <rect>
      <x>-19</x>
      <y>-30</y>
      <width>897</width>
      <height>919</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/borderline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/bear1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_challenge">
    <property name="geometry">
     <rect>
      <x>761</x>
      <y>216</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_communication">
    <property name="geometry">
     <rect>
      <x>761</x>
      <y>344</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_creative">
    <property name="geometry">
     <rect>
      <x>761</x>
      <y>728</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_trust">
    <property name="geometry">
     <rect>
      <x>761</x>
      <y>472</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_unite">
    <property name="geometry">
     <rect>
      <x>761</x>
      <y>600</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_borderline</zorder>
   <zorder>label_main_liberal_option_title</zorder>
   <zorder>label_main_liberal_ment</zorder>
   <zorder>pushButton_right_arrow</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>checkBox_challenge</zorder>
   <zorder>checkBox_communication</zorder>
   <zorder>checkBox_creative</zorder>
   <zorder>checkBox_trust</zorder>
   <zorder>checkBox_unite</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="menu_logo.qrc"/>
 </resources>
 <connections/>
</ui>
